subgraphs/arbitrum-one-bridge/src/handlers/arbitrum/transfers.ts

```diff
@@ -98,15 +102,17 @@ export function handleTransferIn(event: DepositFinalized): void {
 
   // -- TVL
 
-  let inputTokenBalance: BigInt;
-  const erc20 = _ERC20.bind(inputTokenAddress!);
-  const inputTokenBalanceResult = erc20.try_balanceOf(event.address);
-  if (inputTokenBalanceResult.reverted) {
-    log.info("calculate token balance owned by bridge contract reverted", []);
-  } else {
-    inputTokenBalance = inputTokenBalanceResult.value;
+  if (isInputArbToken) {
```

Comment:
Only need to recalculate balance on arb tokens because all the rest are burn/mint, and so no balance, right?

Reply:
Correct